### ISSUE TITLE
Remove ZeroTrust deployment type

### ISSUE THREAD

**Issue by mssalvatore** [guardicore/monkey#1165]
Having a `ztwindows`, `ztdocker`, etc. deployment type adds time and complexity to the build process without providing a lot of value. Remove these deployment types and simplify the build process.

- [x] Remove ability to build different monkey variants from jenkins (0.25d) @mssalvatore 
  - [x] Remove the `variants/ztwindows` directory from the build scripts
  - [x] Remove the `variants/ztdocker` directory from the build scripts
  - [x] Remove code (loop) that looks for different variants to build for Windows
  - [x] Remove code (loop) that looks for different variants to build for Docker
  - [x] Remove variant parameter from jenkins windows build
  - [x] Remove variant parameter from jenkins windows build
- [x] Investigate and repair outdated documentation in Confluence (0d)
- [x] Investigate and make necessary changes to the lambda (0d)

**Comment by mssalvatore** [comment]
The analytics lambda only references the ZT deployments in a lookup table. When requests come in, it looks up the reported deployment type in the table. Removing the ZT deployments from the table would result in incomplete data for older versions of Infection Monkey. 

TLDR; No changes are required for the lambda.
